Sources/RichTextKit/RichTextView_UIKit.swift

```diff
@@ -134,7 +134,21 @@ public extension RichTextView {
         textColor = .label
         setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
         setupInitialFontSize(for: text)
+        
+        let bar = UIToolbar()
```

Comment:
This is nice, but shouldn't be done in `setup` since not everyone may want it.

I'd suggest adding this as some extension or separate function. Can you remove it for now and create a separate PR for it?

Reply:
I was modifying the code for my specific work, will try and fix the PR for it to have only the bug fix, sorry!

Reply:
No worries, just adjust this PR to only contain the subscription and I'll merge this, then we can do the text update fixes after that 👍

Reply:
I will close this PR because my fork is not organized, and create another one for this fix later this week. Sorry for the inconvenience.

Reply:
No worries, thanks anyways! I'll just add the subscription parts and perhaps let's discuss how to best solve the updating of the entire text in a way that doesn't require a delegate 👍